tests/fixtures3.blocks/another-naming-block/__elem/another-naming-block__elem.js

```diff
@@ -0,0 +1,14 @@
+import { decl } from 'bem-react-core';
+
+export default decl({
+    block : 'another-naming-block',
+    elem : 'elem',
+
+    mods : { try : true }
+}, {
+    __dangerouslySetNaming : {
```

Comment:
No, look here https://github.com/bem/bem-react-core/blob/f563168db2444c3a8693b1050730b2503db64748/src/index.js. Point your eyes on the `/* overrides */`. It's the new way to pass settings to the base class. You need to do smth like this:
``` js
export const { decl, declMod } = decls(BaseComponent, inherit({}, { __dengerouslySetNaming : {} }));
```

Also, I think i can make this API workable without `inherit` on the project's level.